ml-conversational-analytic-tool/utils.py

```diff
@@ -0,0 +1,37 @@
+import os
+import ast
+from pathlib import Path
+
+import pandas as pd
+
+EXPORTS_DIR = "exports"
+
+
+def export_to_cvs(export_df: pd.DataFrame, name=None, compound_name=None, suffix="_analysis"):
+    """
+    Export DataFrame into csv file with name constructed based on the given name or compound_name
+    """
+
+    if not os.path.exists(EXPORTS_DIR):
+        os.mkdir(EXPORTS_DIR)
+
+    if name:
+        _, file_extension = os.path.splitext(name)
+        if not file_extension:
+            name = name + ".csv"
+        file = os.path.join(EXPORTS_DIR, name)
+    elif compound_name:
+        file = os.path.join(EXPORTS_DIR, Path(compound_name).stem + suffix + ".csv")
```

Comment:
Because the PR split one command into two, now both commands create a .csv file in an `/exports` directory. To skip code duplication, I have extracted the logic that creates `csv` files into a separate util function. 
The `name` parameter comes directly from (ArgumentParser's) .args.name - > This parameter in both commands is an optional argument.  The validation of whether the argument is provided or not is done in ` export_to_cvs`. The alternative - `compound_name` is always provided - this is the `rawdatafile` name and a specific suffix used for differentiation. Guess that compound name is not appropriate.. probably `alternative` is better.

Reply:
As I see the same functionality exists in githubDataextraction as well.  https://github.com/vmware-labs/ml-conversational-analytic-tool/blob/cc3762d59289668e4d8420bb36fcd7fb8bbd18e3/ml-conversational-analytic-tool/githubDataExtraction.py#L47-L53
do the same. So it may call this util function. 
Related to issue : [Return path to the output files](https://github.com/vmware-labs/ml-conversational-analytic-tool/issues/6)

Reply:
Okay, that clarifies the usage, thanks!
Could we rename this to `alternative-name` as you suggested or `file-name` or `default-name`? I don't think `compound-name` is clear here.

Reply:
done